study.md

```diff
@@ -34,7 +34,9 @@ including class. Why might Array redefine methods included from Enumerable?
 Please give reasons for the methods you list.
 
 ```md
-<!-- your answer here -->
+#map, #reject, #select
+
+I am unsure why Ruby chooses to redefine these methods for Arrays. From what I can tell both the Array and the Enum just yield something to a given block.
```

Comment:
`Array` might redefine methods included from `Enumerable` if custom behavior is desired.  For example, `Array` might redefine `each` to only return elements with even indices on each iteration.